### ISSUE TITLE
Remark attribute in PK column tag is not supported by MariaDB

### ISSUE THREAD

**Issue by janjicandjelka8** [liquibase/liquibase#1793]
## Environment
**Liquibase Version**: 3.6.2

**Liquibase Integration & Version**: springboot 2.0.2, gradle

**Liquibase Extension(s) & Version**: /

**Database Vendor & Version**: MariaDB 10.5.9

**Operating System Type & Version**: Windows 10 Enterprise

## Description
I have defined a changeset that creates table and its columns with pk column "id". I added remark for that column as attribute of column tag, like this:

```java
<!-- table definition/changes -->
  <changeSet id="abc_create_table" author="userABC" labels="labelABC">
    ...
    <createTable tableName="abc" remarks="Table ABC Remark">
      <column name="id" autoIncrement="true" type="BIGINT" remarks="Internal identifier of ABC">
        <constraints primaryKey="true" nullable="false" primaryKeyName="abc_pk"/>
      </column> 
      ...
  </changeSet>
  ...
```

The changeset is run against mariadb on springboot application startup, but the id column results without comment in the mariaDB. 

When I tried to add a remark for that column specifically by using setColumnRemarks tag I got this error:
`1 change have validation failures
setColumnRemarks is not supported on mariadb, 
0_1_0/abc.xml::abc_pk_id_remarks_mariadb::userABC`
I will open a new ticket for supporting setColumnRemarks for mariaDB.

## Steps To Reproduce
* Create a table with PK column that has a remark:

```java
<changeSet id="abc_create_table" author="userABC" labels="labelABC">
    ...
    <createTable tableName="abc" remarks="Table ABC Remark">
      <column name="id" autoIncrement="true" type="BIGINT" remarks="Internal identifier of ABC">
        <constraints primaryKey="true" nullable="false" primaryKeyName="abc_pk"/>
      </column> 
      ...
  </changeSet>
```

* run liquibase update command against MariaDB 10.5.9
* observe there is no remarksfor "id" table column

## Actual Behavior
* No errors in console, id column is created but without remarks/comment.

## Expected/Desired Behavior
* Id column is created with specified remarks/comment.

## Screenshots (if appropriate)
![width=50%](https://user-images.githubusercontent.com/24231732/114176389-5ac65680-993b-11eb-8ad6-fdd7af19d752.png)

## Test Requirements (Liquibase Internal QA)
**Manual Tests**

One MariaDB instance and one MySQL instance is necessary for these tests. Use the attached changelog for these tests.

Note that all structured changelogs (XML, JSON, YML) go through the same code to generate SQL. These tests are changelog format agnostic.

_Verify a update-sql generates SQL for Remark attribute._

`liquibase update-sql --changelog-file LB1429-changelog.xml`

* Generated SQL contains COMMENT attribute for primary key column “pk_column”
* Generated SQL contains COMMENT attribute for table “test_table”
* Generated SQL contains COMMENT attribute for column “test_column”

_Verify update successfully inserts Remark attribute in database._

`liquibase update --changelog-file LB1429-changelog.xml`

* Update command is successful
* Table “test_table” is created with 2 columns “pk_column” and “test_column”
* COMMENT attribute is successfully created
  * “test_table” has comment “Table remark”
  * “pk_column” has comment “Internal identifier of test_table”
  * “test_column” has comment “Column remark”

**Automated Tests**

* No new functional automated tests required for this ticket.
* A new test harness test is required for this ticket.
* Add an XML changelog with one changeset:
  * createTable with primary key column “id” with remarks=”Internal identifier”
  * liquibase-test-harness/src/main/resources/liquibase/harness/change/changelogs
* Add the JSON snapshot for test validation:
  * src/main/resources/liquibase/harness/change/expectedSnapshot/
* Add the generated SQL for test validation:
  * src/main/resources/liquibase/harness/change/expectedSql/mysql
  * src/main/resources/liquibase/harness/change/expectedSql/mariadb

┆Issue is synchronized with this [Jira Story](https://datical.atlassian.net/browse/LB-1429) by [Unito](https://www.unito.io)

**Comment by molivasdat** [comment]
HI @janjicandjelka8 Thanks for reporting this issue.  We won't be doing a fix on 3.6 release but we will look at doing a fix in the 4.x codebase  We will add it to the list of issues that we are processing.

**Comment by php-coder** [comment]
[4.6.2](https://github.com/liquibase/liquibase/releases/tag/v4.6.2) says that this issue is fixed: `[PR#1793] [janjicandjelka8] Fixed the handling of the remarks attribute in a primary key column tag for MariaDB`

**Comment by nvoxland** [comment]
This is a duplicate of #2301 and fixed by #2752. It still seems to be an issue with 4.9.1

I'll close this version to centralize tracking
